kubernetes/manifest.json

```diff
@@ -11,5 +11,5 @@
     "linux",
     "mac_os"
   ],
-  "version": "1.1.1"
+  "version": "1.1.2"
```

Comment:
can be left to 1.1.1, unreleased means it hasn't been shipped in a packaged released yet - even if it's in master already.